substrate/utils/frame/benchmarking-cli/src/overhead/command.rs

```diff
@@ -285,7 +285,7 @@ impl OverheadCmd {
 			log::debug!(target: LOG_TARGET, "Initializing state handler with chain-spec from API: {:?}", chain_spec);
 
 			let source = genesis_builder_to_source();
-			return Ok((GenesisStateHandler::ChainSpec(chain_spec, source), self.params.para_id))
+			return Ok((GenesisStateHandler::ChainSpec(chain_spec, source), self.params.para_id));
```

Comment:
open question - shall we standardize the ';' after return statement?

Reply:
No strong opinions from me. I got a misconfigured formatter, and this got changed and I let it be (not considering a big deal). However I agree we should have a convention. We can live with the existing one until we have enough reasons to change it (for now I don't have a good one).

Changed this to how it was here: [0ba6ecb](https://github.com/paritytech/polkadot-sdk/pull/6450/commits/0ba6ecb244c1ae43bfc6262e2c1143bf3f735410)

Reply:
Agree not a big deal. In future it may become annoying when reviewing PR with larger number of files.